config/environments/development.rb

```diff
@@ -44,7 +44,19 @@
   config.active_storage.service = :local
 
   # Don't care if the mailer can't send.
-  config.action_mailer.raise_delivery_errors = false
+  config.action_mailer.raise_delivery_errors = true
+  config.action_mailer.perform_deliveries = true
+  config.action_mailer.delivery_method = :smtp
```

Comment:
You should rather use letter_opener for the dev env
```suggestion
config.action_mailer.delivery_method = :letter_opener
```

Reply:
I put this in dev env because I can send e-mail confirmation without sending assets to heroku.
Fixed now.